Formula/inko.rb

```diff
@@ -19,13 +19,18 @@ class Inko < Formula
 
   depends_on "coreutils" => :build
   depends_on "rust" => :build
+  depends_on "llvm@15" => :build
 
   uses_from_macos "libffi", since: :catalina
   uses_from_macos "ruby", since: :sierra
 
   def install
-    system "make", "build", "PREFIX=#{prefix}", "FEATURES=libffi/system"
-    system "make", "install", "PREFIX=#{prefix}"
+    ENV['LLVM_SYS_150_PREFIX'] = "/usr/local/Cellar/llvm@15/15.0.7"
+    if OS.mac?
+      ENV['PATH'] = "/usr/local/opt/coreutils/libexec/gnubin:" + ENV["PATH"]
```

Comment:
The Inko buildfile depends on gnu install for the `install -D`lines. This ENV setting makes gnu install come ahead of bsd install in the PATH, but I have a feeling (i.e hope) there is a more idiomatic way to do this?